### DISCUSSION TITLE
Is Inertia dying?

### DISCUSSION THREAD

**Discussion by romsar** [inertiajs/inertia#917] [Help] [answered]
I am afraid to run projects on Inertia.js and face problems of lack of updates in the future. Or should I not worry? 🤔
As I know, now Jonathan is working on Tailwind, Claudio is working on Laravel. So, Inertia has no commits since august, issues and PR's are accumulating.
I don't blame them, Inertia is an open source project, they did perfect job, but without Jonathan and Claudio, I guess, the project will stop developing. 
Any information?

**Comment by reinink** [comment, marked as answer]
Hey there,

No, Inertia.js is not dying. It's an open source project that Claudio and I work on as we have time for it. Prior to us being at Laravel and Tailwind, we had other jobs as well, so nothing really new there.

That said, I have personally been focused on Tailwind the last few months, as this is obviously a big transition for me. The best thing you can do to help the project is try to respond to issues, discussions, and questions on Discord to help beginners with Inertia.

Also worth noting that I gave a talk at Laracon at the beginning of the month on the future of Inertia.

TLDR; Inertia isn't going anywhere — just be aware that development will happen as we have time.
